de/cheatsheet-deep-learning.md

```diff
@@ -144,13 +144,13 @@
 
 **25. Types of gates ― Here are the different types of gates that we encounter in a typical recurrent neural network:**
 
-&#10230; Typen von Gattern - Dies sind die verschiedenen Typen der Gatter die wir in typischen rekurrenten neuronalen Netzen vorfinden:
+&#10230; Typen von Toren - Dies sind die verschiedenen Typen der Tore die wir in typischen rekurrenten neuronalen Netzen vorfinden:
```

Comment:
According to[ german uni](http://www.thi.informatik.uni-frankfurt.de/lehre/clt/sose18/Neuronale%20Netzwerke%20Handout.pdf) "Gatter" was the correct word. So:

**Typen von Gattern - Verschiedene Typen der einzelnen Gattern die man in einem LSTM Block vorfindet:**